crates/sound/rustfmt.toml

```
@@ -0,0 +1,7 @@
+edition = "2018"
```

```suggestion
edition = "2021"
```
maybe? but i believe this should be done in a separate PR so it can land already

---

crate (and others) is in 2018 edition, a follow up PR that updates them to 2021 makes sense